fastlane/Fastfile

```diff
@@ -17,27 +17,28 @@ opt_out_usage
 
 default_platform(:ios)
 
+COCOAPODS_PUSH_WAIT_TIME = 600
+
 pods = [
-  "Amplify.podspec",
-  "AWSPluginsCore.podspec",
-  "CoreMLPredictionsPlugin.podspec",
-  "AWSPredictionsPlugin.podspec",
-  "AmplifyPlugins.podspec"
+  {spec: "Amplify.podspec", sleep: true},
+  {spec: "AWSPluginsCore.podspec", sleep: false},
```

Comment:
`AmplifyPlugins` relies on `AWSPluginsCore`. Doesn't that mean `AWSPluginsCore` should `sleep`?

Reply:
We're sleeping after CoreMLPredictionsPlugin, so that should be good enough, I believe, since CoreMLPredictionsPlugin comes before AmplifyPlugins

Reply:
I see. Since we're using this to model deployment steps, rather than dependencies, I'm OK with leaving it as-is, but let's add a note since it's easy to conflate the two and we don't want to accidentally lose the sleep if we edit this prior to the merge of the CocoaPods "synchronous support" PR.

Reply:
Sounds good. Added a comment: https://github.com/aws-amplify/amplify-ios/pull/379/commits/63042bfff539e02ca614c99b034654dff2555beb#diff-56c5021ef06ac6cea807ae53269b6f4fR24-R25